docs/webhooks/Event-Types.md

```diff
@@ -0,0 +1,20 @@
+# Event Types
+
+Depending on your needs, you may only need a subset of all the events we offer. To make this easy to manage, you can opt in to only the event types you need. You can choose these in our portal, and use them in your code to determine any action you need to take.
+
+This is an excerpt of a request to demonstrate how event types are included in the webhook request.
+
+```json
+{
+[...],
+“eventType”: {
+    “name”: “test”,
+    “version”: 1
+},
+[...]
+}
+```
+
+Within the name of the event is the concept of a namespace to give a more consistent view of what is happening in our system. For example deposit.deposit_settled, which generally refers to a resource that exists on the WealthKernel API, e.g. Parties.
```

Comment:
I think this could be reduced down to just event type names are namespaced, devs reading these docs should be well versed in namespacing.

The second sentence doesn't read quite right to me, what is it trying to explain/demonstrate?

Reply:
It's trying to say that the namespace is going to be one of our API resources. I think I had a placeholder for the event name then came back and rewrote this to add the deposit event but then didn't change the example from Parties to Deposits. Happy to leave it out though.